### PR TITLE
[SDPAP-7408] Remove card_event and card_event_auto

### PR DESCRIPTION
**Pull request by vincent-gao**
### Jira
https://digital-vic.atlassian.net/browse/SDPAP-7408

### Change
clean up card_event and card_event_auto paragraphs

### CODE REVIEW

**Comment by MdNadimHossain** [comment]
Double effort, this work was already there - https://github.com/dpc-sdp/tide_landing_page/pull/196
But that time someone from content team suppose to confirm before it gets merged. I think that work got lost when Meg left. :(

File: tide_landing_page.install
Comment (MdNadimHossain) [inline tide_landing_page.install:2679]: What about these card types, news_listing, card_promotion, card_promotion_auto. These are deprecated as well and if there should be removed.

Comment (vincent-gao) [reply tide_landing_page.install:2679]: hey @MdNadimHossain 
In your branch, have you removed all the relevant fields? We can use your branch instead. I'll close off this branch.

Comment (vincent-gao) [reply tide_landing_page.install:2679]: your branch is ready to merge? @MdNadimHossain

Comment (MdNadimHossain) [reply tide_landing_page.install:2679]: I think I did. But it needs to be updated with the latest develop and also testing required

Comment (MdNadimHossain) [reply tide_landing_page.install:2679]: > your branch is ready to merge? @MdNadimHossain

nope, it is not ready to merge, it was on hold for a long time because someone from the content team suppose to confirm things. So it is long outdated now

Comment (vincent-gao) [reply tide_landing_page.install:2679]: right, let's merge this one specifically for the  https://digital-vic.atlassian.net/browse/SDPAP-7408 ticket.

Comment (MdNadimHossain) [reply tide_landing_page.install:2679]: Cool

**Review by MdNadimHossain — APPROVED** [review]

**Pull request merged**
